src/material/schematics/ng-update/test-cases/v9/hammer-migration-v9.spec.ts

```diff
@@ -59,9 +59,12 @@ describe('v9 HammerJS removal', () => {
         import 'hammerjs';
       `);
 
-      await runMigration();
+      const {logOutput} = await runMigration();
 
       expect(tree.readContent('/projects/cdk-testing/src/main.ts')).not.toContain('hammerjs');
+      expect(logOutput).toContain(
+        'General notice: The HammerJS v9 migration for Angular components is not able to ' +
```

Comment:
It seems like "Angular Components" should be capitalized here since it represents a name (of a specific library) rather than generic "components" used in Angular.

Reply:
good catch. Improving that for sure.